src/vm/arm/asmhelpers.S

```diff
@@ -309,10 +309,12 @@ LOCAL_LABEL(LNullThis):
 //
 // r12 = UMEntryThunk*
 //
-        NESTED_ENTRY UMThunkStub,_TEXT,NoHandler
+        NESTED_ENTRY UMThunkStub,_TEXT,UnhandledExceptionHandlerUnix
         PROLOG_PUSH         "{r4,r5,r7,r11,lr}"
-        push                {r0-r3,r12}
-        PROLOG_STACK_SAVE_OFFSET   r7, #28
+        PROLOG_STACK_SAVE_OFFSET   r7, #8
```

Comment:
@rahku Sorry for late response. libunwind sometimes failed to unwind the stack frame if the frame pointer does not point to the saved frame pointer (discussed in #4581), and thus I used ``PROLOG_STACK_SAVE_OFFSET`` instead of ``PROLOG_STACK_SAVE``